sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -792,9 +792,8 @@ class AccessPolicy(GenAccessPolicy):
     :param start:
         The time at which the shared access signature becomes valid. If
         omitted, start time for this call is assumed to be the time when the
-        storage service receives the request. Azure will always convert values
-        to UTC. If a date is passed in without timezone info, it is assumed to
-        be UTC.
+        storage service receives the request. Azure will always interpret all
+        dates specified to be UTC.
```

Comment:
I think we should not use "Azure" here as it implies the service is interpreting the date as UTC. While true, it is actually because WE are sending a UTC datetime, which is the only thing the service supports. We _could_ convert datetimes to UTC if that were not breaking.

Probably just say "The provided datetime will always be interpreted as UTC." or something like that.